front/src/applications/stdcmV2/components/StdcmEmptyConfigError.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { type FC } from 'react';
+
+import { Tools } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+
+const StdcmEmptyConfigError: FC<unknown> = () => {
+  const { t } = useTranslation('stdcm');
+  return (
+    <div className="d-flex flex-column align-items-center" style={{ marginTop: '5em' }}>
```

Comment:
In the application, we don't use inline style. Create a style file, in `src/styles/scss/applications/stdcmV2`, to move your style here